views/components/Sidebar/SidebarLink.jsx

```diff
@@ -1,13 +1,15 @@
 import React, { forwardRef } from 'react'
+import classNames from 'classnames'
 import { NavLink } from 'react-router-dom'
+
```

Comment:
```suggestion
```

Reply:
this was intentional. It splits the node module imports from local imports.